testing/trino-tests/src/test/java/io/trino/security/TestAccessControl.java

```diff
@@ -380,8 +380,19 @@ public void testTruncateAccessControl()
     public void testUpdateAccessControl()
     {
         assertAccessDenied("UPDATE orders SET orderkey=123", "Cannot update columns \\[orderkey] in table .*", privilege("orders", UPDATE_TABLE));
-        assertThatThrownBy(() -> getQueryRunner().execute(getSession(), "UPDATE orders SET orderkey=123"))
-                .hasMessageContaining("This connector does not support updates");
+        assertAccessDenied("UPDATE orders SET orderkey=123 WHERE custkey < 12", "Cannot select from columns \\[custkey] in table or view .*.default.orders", privilege("orders.custkey", SELECT_COLUMN));
+        assertAccessAllowed("UPDATE orders SET orderkey=123", privilege("orders", SELECT_COLUMN));
```

Comment:
Shouldn't this permission be UPDATE_TABLE?

Reply:
This is mirroring the test for `DELETE FROM orders` without a `WHERE` clause. I believe the intent is to show that `SELECT` privilege is not needed since there is no `WHERE` clause. I can't say if that is correct, but it shows the current behavior of the system.